packages/spectacle/package.json

```diff
@@ -84,36 +76,27 @@
   },
   "wireit": {
     "build": {
-      "dependencies": [
-        "build:lib",
-        "build:dist",
-        "types:create"
-      ]
+      "dependencies": [ "build:lib", "build:dist" ]
     },
     "build:lib": {
-      "dependencies": [
-        "build:lib:esm",
-        "build:lib:cjs"
-      ]
-    },
-    "build:lib:esm": {
-      "command": "nps babel:pkg:lib:esm",
```

Comment:
Minor: We should clean up `package-scripts.js` for things that are no longer used. For example, I didn't see anything for `babel:pkg:lib`